orchestrator/utils/src/cosmos/send.rs

```diff
@@ -92,10 +92,10 @@ pub async fn send_ethereum_claims(
             creator: our_address.to_string(),
             event_nonce: power_change.event_nonce,
             block_height: downcast_uint256(power_change.block_height).unwrap(),
-            token_contract: power_change.erc20.to_string(),
-            amount: power_change.amount.to_string(),
-            cosmos_receiver: power_change.destination,
-            ethereum_sender: power_change.sender.to_string(),
+            token_contract: power_change.token.to_string(),
+            amount: power_change.shares.to_string(),
```

Comment:
How do shares map to amount?

Reply:
its 1:1 basically amount of stake change